app/models/round.rb

```diff
@@ -185,7 +185,7 @@ def score_taking_done?
     competitors_live_results_entered == total_accepted_registrations
   end
 
-  def has_undef_tl?
+  def undef_tl?
```

Comment:
I'm personally fine with this, but in the interest of avoiding abbreviations what about `time_limit_undefined?`